src/Mail/SmtpMailer.php

```diff
@@ -201,6 +201,6 @@ protected function read()
  *
  * @author     David Grudl
  */
-class SmtpException extends \Exception
+class SmtpException extends SendFailedException
```

Comment:
I think it should be the other way around. `SendFailedException` should extend from `SmtpException`